client/client-multi/client-c/src/kaa/kaa_channel_manager.c

```diff
@@ -96,10 +100,12 @@ kaa_error_t kaa_channel_manager_create(kaa_channel_manager_t **channel_manager_p
     (*channel_manager_p)->transport_channels      = kaa_list_create();
     KAA_RETURN_IF_NIL((*channel_manager_p)->transport_channels, KAA_ERR_NOMEM);
 
-    (*channel_manager_p)->kaa_context             = context;
-    (*channel_manager_p)->sync_info.request_id    = 0;
-    (*channel_manager_p)->sync_info.is_up_to_date = false;
-    (*channel_manager_p)->logger                  = context->logger;
+    (*channel_manager_p)->kaa_context                   = context;
```

Comment:
Avoid using ugly constructs, like `(*channel_manager_p)`

Just declare above pointer-to-object and use it